listenbrainz/webserver/static/js/src/stats/UserEntityCharts.tsx

```diff
@@ -22,17 +23,18 @@ export type UserHistoryState = {
   maxListens: number;
   startDate: Date;
   loading: boolean;
+  calculated: boolean;
 };
 
-export default class UserHistory extends React.Component<
-  UserHistoryProps,
-  UserHistoryState
+export default class UserEntityCharts extends React.Component<
```

Comment:
this is really nitpicky, but I think this should be `UserEntityChart`, the convention is that React components are singular.

Reply:
Fixed